src/components/organisms/UiMultipleAnswer/_internal/UiMultipleAnswerItem.vue

```diff
@@ -101,80 +86,129 @@ const props = defineProps({
+    default: () => ({}),
+  },
   /**
    * Use this props to pass attrs for info UiButton.
    */
   buttonInfoAttrs: {
     type: Object,
-    default: null,
+    default: () => ({}),
   },
   /**
-   * Use this props to pass attrs for label UiText.
+   * Use this props to pass attrs for info label element.
    */
-  textLabelAttrs: {
+  labelInfoAttrs: {
     type: Object,
-    default: () => ({ tag: 'span' }),
+    default: () => ({}),
   },
   /**
-   * Use this props to pass attrs for info UiIcon.
+   *  Use this props to pass attrs for info UiIcon.
    */
   iconInfoAttrs: {
     type: Object,
     default: () => ({ icon: 'info' }),
   },
-});
-interface DefaultProps {
-  textLabelAttrs: {
-    tag: HTMLTag;
-    [key:string]: unknown;
-  };
-  iconInfoAttrs: {
-    icon: Icon;
-    [key:string]: unknown;
-  };
-}
-const defaultProps = computed<DefaultProps>(() => ({
-  textLabelAttrs: {
-    tag: 'span',
-    ...props.textLabelAttrs,
-  },
-  iconInfoAttrs: {
-    icon: 'info',
-    ...props.iconInfoAttrs,
+  /**
+   * Use this props to pass labels inside component translation.
+   */
+  translation: {
+    type: Object,
+    default: () => ({ info: 'What does it mean?' }),
   },
-}));
+});
 const emit = defineEmits([ 'update:modelValue' ]);
 const isCheckbox = computed(() => (Array.isArray(props.modelValue)));
 const component = computed(() => (isCheckbox.value ? UiCheckbox : UiRadio));
 const componentName = computed<ComponentName>(() => (isCheckbox.value ? 'ui-checkbox' : 'ui-radio'));
-const errorClass = computed(() => ([
-  props.invalid ? `${componentName.value}--has-error` : '',
-  { 'ui-multiple-answer-item--has-error': props.invalid },
-]));
-function focusExplication(event: KeyboardEvent) {
+const errorClass = computed(() => (props.invalid
+  ? [
+    `${componentName.value}--has-error`,
+    'ui-list-item--has-error',
+  ]
+  : []));
+
+function handleInfoFocus(event: KeyboardEvent) {
   if (event.key !== 'ArrowRight') return;
-  const el = event.target as HTMLInputElement;
-  const explicationButton: HTMLInputElement | null | undefined = el.closest(`.${componentName.value}`)?.querySelector('.ui-multiple-answer-item__explication');
-  if (explicationButton) {
+  const input = event.target as HTMLInputElement;
+  const info: HTMLElement | null | undefined = input
+    .closest('.ui-multiple-answer-item')
+    ?.querySelector('.ui-multiple-answer-item__info');
+  if (info) {
     event.preventDefault();
-    focusElement(explicationButton);
+    focusElement(info);
   }
 }
-function unfocusExplication(event: KeyboardEvent) {
+function handleInfoUnfocus(event: KeyboardEvent) {
   if (event.key !== 'ArrowLeft') return;
-  const el = event.target as HTMLInputElement;
-  const answerInput: HTMLInputElement | null | undefined = el.closest(`.${componentName.value}`)?.querySelector('input');
-  answerInput?.focus();
+  const info = event.target as HTMLElement;
+  const input: HTMLInputElement | null | undefined = info
```

Comment:
What's going on over there? 😄 In one line there's casting `KeyboardEvent` `.target` type to `HTMLElement` (why?) and in the next line, below, this newly created variable with casted `HTMLElement` type is assigned to new variable with declared as `HTMLInputElement | null | undefined`? 😮 🤯 😁